chalicelib/checks/badge_checks.py

```diff
@@ -177,29 +177,32 @@ def yellow_flag_biosamples(connection, **kwargs):
         diff_auth = False
         ploidy = False
         bccs = result.get('cell_culture_details', [])
-        for bcc in bccs:
-            for item in [
-                'culture_harvest_date', 'doubling_number', 'passage_number', 'culture_duration', 'morphology_image'
-            ]:
-                if not bcc.get(item):
-                    messages.append('Biosample missing {}'.format(item))
-            if bcc.get('karyotype'):
-                karyotype = True
-            for protocol in bcc.get('authentication_protocols', []):
-                protocol_item = ff_utils.get_metadata(protocol['@id'], key=connection.ff_keys)
-                auth_type = protocol_item.get('protocol_classification')
-                if not karyotype and auth_type == 'Karyotype Authentication':
+        if not bccs:
```

Comment:
Does this mean that things like tissues and maybe primary cells that aren't actually cultured will get flagged?  

I'm not sure what goes on in this function 'compare_badges_and_messages' so maybe that is where non relevant Biosources get filtered?